docs/MO_DG/prepare_model/convert_model/Converting_Model_General.md

```diff
@@ -139,7 +139,9 @@ Framework-agnostic parameters:
                         multiline mode.
   --transformations_config TRANSFORMATIONS_CONFIG
                         Use the configuration file with transformations
-                        description.
+                        description. File can be specified as relative path 
+                        from the current directory, as absolute path or as a 
+                        relative path from the mo root directory
```

Comment:
Please remove "File can be specified as relative path 
                        from the current directory, as absolute path or as a 
                        relative path from the mo root directory"
I don't think it is super necessary thing that we are proud of.